lib/web/apiserver.go

```diff
@@ -2865,6 +2868,43 @@ func (h *Handler) getClusters(w http.ResponseWriter, r *http.Request, p httprout
 	return out, nil
 }
 
+type GetClusterInfoResponse struct {
+	ui.Cluster
+	IsCloud bool `json:"isCloud"`
+}
+
+// getClusterInfo returns the information about the cluster in the :site param
+//
+// GET /v1/webapi/sites/:site/info
+//
+// Successful response:
+//
+//	{"name": "localhost","lastConnected": "RFC3339 time","status": "online","publicURL": "localhost:3080","authVersion": "17.0.0-dev","proxyVersion": "17.0.0-dev", "isCloud": true}
```

Comment:
nit: I would remove the response reference from here and if there is particular field that we want to showcase, just explain that only. Because this risks going out of sync if we end up changing response shape.

Reply:
I agree, but I added it because I saw that in other functions.

Looking at this file's history, it looks like we stopped with this practice, so I removed all but the first godoc line. Thanks.